src/Standards/Generic/Tests/PHP/DisallowShortOpenTagUnitTest.3.inc

```diff
@@ -1,6 +1,6 @@
 // Test warning for when short_open_tag is off.
 
-Some content <? echo $var; ?> Some more content
+Some content <? echo $var; ?> Some more content after PHP code
```

Comment:
I made the text longer so that the following line is exercised by the tests: https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/src/Standards/Generic/Sniffs/PHP/DisallowShortOpenTagSniff.php#L160

Reply:
I think it may be better to have a separate test for that. I.e. leave the original tests as-is and add a second line with a longer text (which may even say in the text that the line is longer to test the snippet clipping).

Reply:
Thanks for your suggestion. I just pushed a new commit to implement it.